index.js

```diff
@@ -22,12 +25,15 @@ const interpolate = paraphrase(/\${([^{}]*)}/g, /%{([^{}]*)}/g, /{{([^{}]*)}}/g)
  * @param {Object<Object>}   options.translations JSON compliant object
  * @param {Object<String>}   [options.$scope]     Root string to be use for looking for translation keys
  * @param {Object<Function>} [options.missing]    Method to call when key is not found
+ * @param {Object<Function>} [options.empty]      Method to call when value is empty
  */
 class I18n {
-    constructor({translations, $scope, missing} = {translations: {}, $scope: undefined, missing: undefined}) {
+    constructor({translations, $scope, missing, empty} = {translations: {}}) {
```

Comment:
```suggestion
    constructor({translations, $scope, missing, empty} = {translations: {}, missing: () => undefined, empty: () => undefined}) {
```

Reply:
Not exactly, you meant:
```suggestion
    constructor({translations, $scope, missing = () => null, empty = () => null} = {translations: {}}) {
```

But in actuality if someone passes a non function the while default system breaks. This way we're guaranteed onmiss and onempty are functions

Reply:
Yeah, my bad, meant to use it there.